hack/make-aggregator-pki.sh

```diff
@@ -7,6 +7,21 @@ set -o pipefail
 rm -rf data/grafana-aggregator
 
 mkdir -p data/grafana-aggregator
+
 openssl req -nodes -new -x509 -keyout data/grafana-aggregator/ca.key -out data/grafana-aggregator/ca.crt
-openssl req -out data/grafana-aggregator/client.csr -new -newkey rsa:4096 -nodes -keyout data/grafana-aggregator/client.key -subj "/CN=development/O=system:masters"
-openssl x509 -req -days 365 -in data/grafana-aggregator/client.csr -CA data/grafana-aggregator/ca.crt -CAkey data/grafana-aggregator/ca.key -set_serial 01 -sha256 -out data/grafana-aggregator/client.crt
+openssl req -out data/grafana-aggregator/client.csr -new -newkey rsa:4096 -nodes -keyout data/grafana-aggregator/client.key \
+  -subj "/CN=development/O=system:masters" \
+  -addext "extendedKeyUsage = clientAuth"
+openssl x509 -req -days 365 -in data/grafana-aggregator/client.csr -CA data/grafana-aggregator/ca.crt -CAkey data/grafana-aggregator/ca.key \
+  -set_serial 01 \
+  -sha256 -out data/grafana-aggregator/client.crt \
+  -copy_extensions=copyall
+
+openssl req -out data/grafana-aggregator/server.csr -new -newkey rsa:4096 -nodes -keyout data/grafana-aggregator/server.key \
```

Comment:
Adds a new certificate for local testing so that the `apiserver` cmd used to launch aggregated servers need not auto-generate it's TLS cert with another throwaway CA.

This allows us to say, set `CABundle` on an APIService and validate the end-to-end aggregation workflow without skipping TLS.